pyquil/tests/test_noise.py

```diff
@@ -134,7 +134,7 @@ def test_decoherence_noise():
 
     # verify that gate names are translated
     new_prog = apply_noise_model(prog, m3)
-    new_gates = _get_program_gates(new_prog)
+    _get_program_gates(new_prog)
```

Comment:
`_get_program_gates` doesn't appear to be side-effecty, nix? or else maybe the intention was to do something with `new_gates`?